### ISSUE TITLE
Add 5 posts for Narmok introduction and our guidelines

### ISSUE THREAD

**Issue by parhamin2010** [parhamin2010/narmok#10]
Do a research and report it here then we decide to write in what topics.

**Comment by Sherwin83** [comment]
start to do research

**Comment by Sherwin83** [comment]
we can write about :
1-buying guides
2-coming soon
3-customers profile
4-news(fashion/new way of transfer/magic offer....)

**Comment by parhamin2010** [comment]
Search about first one please. See what you can provide to the users.

**Comment by Sherwin83** [comment]
Ok , i'll do that.

**Comment by Sherwin83** [comment]
We can write about smart watch buying guide and categorize them according to the best screen/best value/lowest price/best quality or ....

And we can write about anything else

**Comment by parhamin2010** [comment]
No, please write about the how user can buy a product from website.

**Comment by Sherwin83** [comment]
[  ] Sign in/log in
[  ] Select the product you want
[  ] Select the amount of shares you want to buy
[  ] Click add to cart
[  ] If you have coupon code add it
[  ] Click on proceed to checkout
[  ] Fill in the form and then click on place order

We can fallow this structure.
All of them place in website with guide picture.

**Comment by Sherwin83** [comment]
I can provide that tomorrow or tonight. 
And i think this thing should add to blog page in banner on top or in bottom of the home page.

**Comment by parhamin2010** [comment]
As we discussed, we already have one way. 

> Show both of them and say we have two ways for buying products.

**Comment by parhamin2010** [comment]
@Sherwin83 please proceed on this too. we need more 4 blogs.

**Comment by Sherwin83** [comment]
Sure , i'll work on it.

**Comment by Sherwin83** [comment]
This task is done and it's on blog page and descript it on a post. please review.
![sag1](https://github.com/parhamin2010/narmok/assets/131457314/24dcb36f-e1b0-4c3f-8c76-0834200672c4)

**Comment by parhamin2010** [comment]
After write 5 article this task will be close.
